charts/stolon/templates/components/keeper/statefulset.yaml

```diff
@@ -112,6 +112,52 @@ spec:
         volumeMounts:
         - name: data
           mountPath: /stolon/data
+      {{- if .Values.exporter.enabled }}
+      - name: exporter
+        image: {{ .Values.exporter.image.repository }}:{{ .Values.exporter.image.tag }}
+        imagePullPolicy: {{ .Values.exporter.image.pullPolicy }}
+        args:
+        {{- if .Values.exporter.disableDefaultMetrics }}
+        - "--disable-default-metrics"
+        {{- end }}
+        {{- if .Values.exporter.disableSettingsMetrics }}
+        - "--disable-settings-metrics"
+        {{- end }}
+        env:
+        {{- if .Values.exporter.datasourceSecret }}
+        - name: DATA_SOURCE_NAME
+          valueFrom:
+            secretKeyRef:
+              name: {{ .Values.exporter.datasourceSecret.name }}
+              key: {{ .Values.exporter.datasourceSecret.key }}
+        {{- else }}
```

Comment:
I think it is useless option. We'll use just `DATA_SOURCE_URI` env variable

Reply:
resolved